lib/hotel_system.rb

```diff
@@ -0,0 +1,49 @@
+
+module Hotel
+  class HotelSystem
+    attr_reader :rooms, :reservations
+    def initialize
+      @rooms = (1..20).to_a 
+      @reservations = []
+    end
+    
+    def list_rooms 
+      return @rooms.dup
+    end
+
+    def available_rooms(date_range)
```

Comment:
whoops, you left an extra `available_rooms` method lying around. The one below this overwrites the functionality here, but you should delete this from your code.